infrastructure/cdn-in-a-box/traffic_portal/run.sh

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -ex
+env
+
+export TP=/opt/traffic_portal/
+
+envvars=( TO_PORT )
+for v in ${envvars[*]}; do
+	val=${!v}
+	[[ -z $val ]] && echo "$v is unset" && exit 1
+done
+
+mkdir -p /etc/traffic_portal/conf/
+mkdir -p /etc/pki/tls/private
+mkdir -p /etc/pki/tls/certs
+
+cat >/etc/traffic_portal/conf/config.js <<-TPCONF
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Do we really need the ASF header in this generated config file?

Reply:
no -- we don't..